### PR TITLE
Update pr-wranglers.md

### PR DESCRIPTION
**Pull request by a-mccarthy**
Based on a discussion in the sig docs meeting today, I added a brief note to the pr wrangler pages to give more context for wranglers on reviewing language PRs. Happy to get some feedback on anything else to include. (i didn't see this written anywhere else, but if it already exists, please feel free to close this PR)

cc: @natalisucks @reylejano @seokho-son

### CODE REVIEW

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ Each day in a week-long shift as PR Wrangler:
     - Using style fixups as good first issues is a good way to ensure a supply of easier tasks
       to help onboard new contributors.
 
+{{< note >}}
+PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, its often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more). 
+{{< /note >}}
```
Comment (tengqm) [inline content/en/docs/contribute/participate/pr-wranglers.md:50]: Please wrap this long line as we did for other paragraphs.

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ Each day in a week-long shift as PR Wrangler:
     - Using style fixups as good first issues is a good way to ensure a supply of easier tasks
       to help onboard new contributors.
 
+{{< note >}}
+PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, its often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more).
```
Comment (reylejano) [inline content/en/docs/contribute/participate/pr-wranglers.md:49]: non-blocking nit: `its` is possessive
```suggestion
PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, it's often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more). 
```

**Comment by reylejano** [comment]
Preview: https://deploy-preview-45443--kubernetes-io-main-staging.netlify.app/docs/contribute/participate/pr-wranglers/

File: content/en/docs/contribute/participate/pr-wranglers.md (hunk 2 above)
Comment (seokho-son) [inline content/en/docs/contribute/participate/pr-wranglers.md:49]:
```suggestion
PR wrangler duties do not apply to localization PRs (non-English PRs). Localization teams have their own processes and teams for reviewing their language PRs. However, if you have time during your shift, it's often helpful to ensure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long-running PRs (ones opened more than 6 months ago and have not been updated in a month or more).
```
Hi @a-mccarthy 
I think the phrase "PR Wrangler duties apply to English language PRs" could be clarified further. The reason is that the current duties of a PR Wrangler apply not only to English language PRs but also to PRs that are independent of language (for example, workflow, site-configuration, scripts, etc.). Therefore, stating that these duties do not apply to localization PRs might make it clearer.

I'm not sure about the blog area. I haven't checked whether blog PRs are an exception to PR Wrangler duties or not.

**Review by seokho-son — COMMENTED** [review]
Thanks @a-mccarthy !
I have a minor comment. :)

File: content/en/docs/contribute/participate/pr-wranglers.md (hunk 2 above)
Comment (a-mccarthy) [reply content/en/docs/contribute/participate/pr-wranglers.md]: good point @seokho-son, i've added in your suggestions.

**Comment by reylejano** [comment]
/lgtm

I can approve if there's a second lgtm

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**